docker-compose.yml

```diff
@@ -53,12 +44,6 @@ services:
       - ./.envs/.local/.frontend
     environment:
       - OVERRIDE_API_PROXY
-      - NODE_PATH=/home/node_install/node_modules
-
-  mailhog:
-    image: mailhog/mailhog:v1.0.0
-    ports:
-      - "8025:8025"
```

Comment:
Why was mailhog removed? I thought we needed this to test mail locally

Reply:
Local development now uses the console EmailBackend, so messages are just printed to stdout in the Django container:

https://github.com/deardurham/dear-petition/blob/5f2bce75d5efc22761ffa07de0aaa076777aa3dd/config/settings/local.py#L35